app/controllers/chargeback_controller.rb

```diff
@@ -725,6 +725,21 @@ def get_tags_all(category)
     classification.entries.each { |e| @edit[:cb_assign][:tags][e.id.to_s] = e.description } if classification
   end
 
+  def get_sub_tenants(tenant, parent)
```

Comment:
@aljesusg can we move this method to model ? and rename it somehow 'build_tenant_tree' or somehow it that way ?
 because to reason for this method is build Hash with tree structoru of tenants, right ? what do you think ?